api/src/infrastructure/docker.rs

```diff
@@ -450,6 +485,12 @@ impl DockerInfrastructure {
             labels.insert(REPLICATED_ENV_LABEL, replicated_env);
         }
 
+        options.volumes(
```

Comment:
The suggestion above might save you an iter/collect combination here.

Reply:
options.volumes consumes Vec<&str> and host_config_binds are created with format!() which is an owned string. One round of iter+collect is needed to make it to Vec<&str>.